### PR TITLE
Moved API calls to meter readings and payments to backend

### PR DESCRIPTION
**Pull request by julianrkung**
**Note**: File is blocked by https://github.com/calblueprint/meepanyar/pull/26 . Do not merge until this is unblocked.

**Changelist**:
- Moved API calls for customers / meter readings / payments to backend (used to be in frontend). Associated PR for backend: https://github.com/calblueprint/meepanyar-node/pull/3 
- Added a patch and associated README that should be run after new `airtable-schema-generator` runs that changes `customerIds` to `customers` on the site schema
- `refreshSiteData` no longer takes in parameters because it never used the parameters in the first place.

### CODE REVIEW

File: src/lib/airtable/patches/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Overview
```
Comment (jenmhoang) [inline src/lib/airtable/patches/README.md:1]: this is so neat you are the best

Comment (jenmhoang) [reply src/lib/airtable/patches/README.md:1]: Should we also add preservations of non-generated functions in request.js that would be overwritten upon re-generation?

File: src/lib/airtable/schema.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const Columns = {
 	"Sites": {
 		name: {name:`Name`, type:`text`},
 		userIds: {name:`Users`, type:`foreignKey-many`},
-		customerIds: {name:`Customers`, type:`foreignKey-many`},
+		customers: {name:`Customers`, type:`custom-object`}, // Modified via patch.
 		financialSummarieIds: {name:`Financial Summaries`, type:`foreignKey-many`},
 		tariffPlanIds: {name:`Tariff Plans`, type:`foreignKey-many`},
 		field6: {name:`Field 6`, type:`text`},
```
Comment (jenmhoang) [inline src/lib/airtable/schema.js:35]: looks like an unnecessary extra field was added into the Airtable and included in the generation

**Review by jenmhoang — APPROVED** [review]
your genius is unmatched

File: src/lib/airtable/patches/README.md (hunk 1 above)
Comment (julianrkung) [reply src/lib/airtable/patches/README.md:1]: Good idea. I'll do that in another PR that'll also remove the extra airtable field you included in the other comment

File: src/lib/airtable/schema.js (hunk 2 above)
Comment (julianrkung) [reply src/lib/airtable/schema.js:35]: Good catch. I'll remove it in another PR that'll also add a patch for the non-generated functions

**Pull request merged**